docs/programming/parallel-programming-fundamentals.md

```diff
@@ -4,9 +4,9 @@ Parallel commputation can often be a bit more complex compared to
 standard serial applications. This page will explore these differences and describe how parallel programs work in general.
```

Comment:
Under the section `Work Sharing Directive: omp do`, we should highlight partial_Sum and total_Sum in code syntax. 

in the part: 
```
Now we must join our threads. To do this we must use a critical directive to create a thread safe section of code. We do this with the !$OMP CRITICAL directive. Lastly we add partial_Sum to total_Sum and print out the result outside the parallel section of code.
```

Reply:
Did you link to the wrong document? I can't find this anywhere in parallel-programming-fundamentals.md.

Reply:
Yes, apologies it is in `programming/OpenMP-Fortran.md`